### PR TITLE
Delete aggregator layer

### LINKED ISSUE #280: [Feature Contribution]: Code cleanup to delete Aggregator project and references
### What happened?

Based on the [ADR](https://github.com/Green-Software-Foundation/carbon-aware-sdk/pull/275) , we no longer need the Aggregator layer in the SDK. The WebAPI and CLI directly call the GSF handlers making the aggregator layer obsolete.

Delete the Aggregator project along with any remnant references in the code and update the documentation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Feature Commitment

- [X] I commit to contributing this feature as a PR and working with the GSF to merge this feature into the Carbon Aware SDK.

### PR DESCRIPTION
**Pull request by pritipath**
Issue Number: #280

## Summary
This PR deletes the Aggregator code and any references to it. Documentation is updated as well.

## Changes

- List of comprehensive changes

## Checklist

- [ ] Local Tests Passing?
- [ ] CICD and Pipeline Tests Passing?
- [ ] Added any new Tests?
- [ ] Documentation Updates Made?
- [ ] Are there any API Changes? If yes, please describe below.
- [ ] This is not a breaking change. If it is, please describe it below.

## Are there API Changes?
If yes, what are the expected API Changes? Please link to an API-Comparison workflow with the API Diff.

## Is this a breaking change?
If yes, what workflow does this break? 

## Anything else?
Other comments, collaborators, etc.
> Please follow [GitHub's suggested syntax](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) to link Pull Requests to Issues via keywords

This PR Closes #280

### CODE REVIEW

File: src/CarbonAware.WebApi/test/integrationTests/CarbonAware.WebApi.IntegrationTests.csproj [hunk 1]
```diff
@@ -28,6 +28,7 @@
     <ProjectReference Include="..\..\..\CarbonAware.DataSources\CarbonAware.DataSources.WattTime\mock\CarbonAware.DataSources.WattTime.Mocks.csproj" />
     <ProjectReference Include="..\..\..\CarbonAware.DataSources\CarbonAware.DataSources.ElectricityMaps\mock\CarbonAware.DataSources.ElectricityMaps.Mocks.csproj" />
     <ProjectReference Include="..\..\src\CarbonAware.WebApi.csproj" />
+	<ProjectReference Include="..\..\..\CarbonAware.DataSources\CarbonAware.DataSources.Registration\CarbonAware.DataSources.Registration.csproj"   />
```
Comment (JenMadiedo) [inline src/CarbonAware.WebApi/test/integrationTests/CarbonAware.WebApi.IntegrationTests.csproj:31]: 🐑 indentation

File: src/CarbonAware.WebApi/test/unitTests/TestsBase.cs [hunk 2]
```diff
@@ -24,17 +21,12 @@ protected TestsBase()
...
+        handler.Setup(x => x.GetEmissionsDataAsync(It.IsAny<string[]>(), null, null))
             .Callback((string[] location, DateTimeOffset? start, DateTimeOffset? end) =>
             {
-                aggregator.Setup(x => x.GetEmissionsDataAsync(It.IsAny<CarbonAwareParameters>()))
-                    .Callback((CarbonAwareParameters parameters) =>
-                    {
-                        parameters.SetRequiredProperties(PropertyName.MultipleLocations);
-                        parameters.Validate();
-                    }).ReturnsAsync(It.IsAny<List<CarbonAware.Model.EmissionsData>>());
+                Assert.NotNull(location);
```
Comment (JenMadiedo) [inline src/CarbonAware.WebApi/test/unitTests/TestsBase.cs:28]: Love how clean this is now

**Review by JenMadiedo — APPROVED** [review]
LGTM, I would just update the description to finish filling ou the PR template and adding a link to the commits only relevant to this PR

**Review by Kpopovs1 — APPROVED** [review]
🚀

**Review by juzuluag — APPROVED** [review]
Nice! 👍

**Review by vaughanknight — APPROVED** [review]
Looks great, thanks for the update.

**Pull request merged**